include/swift/Option/Options.td

```diff
@@ -185,6 +188,9 @@ def I : JoinedOrSeparate<["-"], "I">, Flags<[FrontendOption, ArgumentIsPath]>,
 def I_EQ : Joined<["-"], "I=">, Flags<[FrontendOption, ArgumentIsPath]>,
   Alias<I>;
 
+def M : Joined<["-"], "M">, Flags<[FrontendOption]>,
```

Comment:
This should be JoinedOrSeparate. I don't think we have any Joined-only arguments except those that end in `=`.